src/ansible_compat/runtime.py

```diff
@@ -59,13 +60,21 @@ def __init__(
         :param isolated: Assure that installation of collections or roles
                          does not affect Ansible installation, an unique cache
                          directory being used instead.
+        :param version: Minimal version of Ansible required. If not found,
+                        a RuntimError exception is raised.
         """
         self.project_dir = project_dir or os.getcwd()
         self.isolated = isolated
         if isolated:
             self.cache_dir = get_cache_dir(self.project_dir)
         self.config = AnsibleConfig()
 
+        if version and packaging.version.Version(version) > self.version:
```

Comment:
"if version" is semantically incorrect and may result in an unexpected control flow. Like if it's `0`.
The correct way of the comparison with `None` is to have an identity check like `if version is not None`.